app/furniture/marketplace/order.rb

```diff
@@ -12,6 +12,7 @@ class Order < Record
     has_many :products, through: :ordered_products, inverse_of: :orders
 
     has_encrypted :delivery_address
+    attribute :delivery_window, ::Marketplace::Delivery::WindowType.new
```

Comment:
I feel like this should work with `delegate :window, to: :delivery, prefix: true`

Reply:
Note! It does *not* because:

1. `becomes` does not re-initialize the data from the database, so the `attribute` needs to be pulled initially!

Reply:
This *will* work when we make a `marketplace_deliveries` table instead of using the `becomes` hack.